Modix.Bot/Modules/ReplModule.cs

```diff
@@ -35,8 +35,11 @@ public class ReplModule : ModuleBase
         // 1024 is the maximum field embed size
         // minus 3 for the start codeblocks
         // minus 3 for the end codeblocks
+        // minus 1 for the first newline after the ```cs
+        // minus 2 for 'cs'
+        // minus 1 for the next newline after the code
         // THIS LIMIT IS NOT ARBITRARY
-        private const int MaxReplSize = 1_024 - 3 - 3;
+        private const int MaxReplSize = 1_024 - 3 - 3 - 1 - 2 - 1;
```

Comment:
This is getting a little hard to parse. Would it be better to just set it to 1014 with a comment?

Reply:
hard to parse? i think each comment in order of why we're removing chars and then removing it in the code is fine

Reply:
Each comment explains clearly why it should be set to 1014, doing it this way is extremely verbose, which is what I think @Scott-Caldwell was getting at.

Reply:
I'm ok with composing it like this. It is a bit awkward to look at but the comment provides the necessary clarity.